core/.changelog.d/4302.added

```diff
@@ -0,0 +1,4 @@
+[T3T1] Add page counter to paginated blobs.
+[T3T1] Add cancel button to individual pages of a blob.
+[T3T1] Add margin to the first page of a blob.
+[T3B1] Add "continue" button after scrolling through the whole blob.
```

Comment:
each of those needs its own file (`4302.added`, `4302.added.1`, `4302.added.2` etc)

(check the output of `./CHANGELOG.unreleased` in core dir

Reply:
I wonder why do we need such granularity in the changelog though. Does it mean anything for the customer?